docs/docsite/rst/network_aggregate_resources.rst

```diff
@@ -0,0 +1,95 @@
+*********************************
+Aggregate Resources in Networking
+*********************************
+
+.. contents:: Topics
+
+This section explores how and when `Aggregate Resources` can be used within Ansible Networking.
+
+This document is part of a collection on Networking. The complete list of guides can be found at :ref:`Network Guides <guide_networking>`.
+
+Overview
+========
+
+Consider the case if you wanted to ensure a set of vlans are present on a switch. Although at a high level thia sounds like a simple request, there in a slight nuance:
+
+* Should these vlans be in *addition* to the existing configuration?  - *additive*
+* Should these vlans be the *only* ones present? - *aggregate*
+
+
+.. versionadded:: 2.4
+
+   The ``aggregate:`` option has been added in Ansible 2.4 and is available in certain modules, see the modules documentation to see if the feature is available.
+
+Additive resources
+===================
+
+Continuing our vlan example, the following task ensure that vlans ``1``, ``2`` and ``3`` are in the states specified in the task, in `addition` to any existing vlans.
+
+This task *will not* change any vlans already configured on the switch, apart from the ones specified in the task.
+
+.. code-block:: yaml
+
+   - name: configure vlans neighbor in addition to existing
+     net_vlan:
+       vlan_id: “{{ item.vlan_id }}”
+       name: “{{ item.name }}”
+       state: “{{ item.state | default(‘active’) }}”
+     with_items:
```

Comment:
That's a good question. Originally I wanted to show the different ways of doing things, though that may just confuse the matter.

I've added a `FIXME` in the docs.